### ISSUE TITLE
does not handle utf8 correctly

### ISSUE THREAD

**Issue by ghost** [noir-clojure/noir#56]
To test, define a basic page that accepts a GET parameter.

``` clojure
(defpage "/:x" {:keys [x]} x)
```

Then make a request to http://localhost:8080/Ǿ

Expected: Ǿ
Actual: «æ

Using the following Compojure code I get the expected result.

``` clojure
(ns test
  (:use compojure.core, ring.adapter.jetty)
  (:require [compojure.route :as route]))
(defroutes main-routes (GET "/:x" [x] x))
(run-jetty main-routes {:port 8080})
```

**Comment by ibdknox** [comment]
fixed in 1.2.2-SNAPSHOT

**Comment by ghost** [comment]
Thanks Chris.

The test for this particular UTF8 problem did not really expose my actual problem.
I still had problems with UTF8 when submitting UTF8 characters in HTML forms. The following JVM option fixes it.

`:jvm-opts ["-Dfile.encoding=utf-8"]`
